### DISCUSSION TITLE
Support for 'encv', 'enca' box

### DISCUSSION THREAD

**Discussion by kosh271** [essential61/mp4analyser#31] [Ideas]
If the .mp4 file to analyze contains the 'encv' box type instead of 'avc1', the parser fails to parse the file (and provides no output other than an error message at the terminal).  To fix the issue, I've added **'EncvBox = Avc1Box'** in the _non_iso.py_ just below 'Avc4Box = Avc3Box = Avc2Box = Avc1Box' in the non_iso.py file.

The 'enca' box is handled (and fixed) much the same way - adding **'EncaBox = Mp4aBox'** just below 'Ac_3Box = Ec_3Box = Mp4aBox' in the non_iso.py file.

Attached is an example of a file with the 'encv' box.  Reference standard ISO/IEC 14496-12:2005, sec. 8.45 for more information.

https://user-images.githubusercontent.com/37758309/154688615-73cbbce3-737e-40ee-9a41-c27740833018.mp4

**Comment by essential61** [comment]
When did you clone the repo? I did commit this same change in Dec 21-23.

**Comment by kosh271** [comment]
My apologies, I grabbed the last release and didn't look to see if it had
been added in the repo.

That is completely my fault. I apologize for not looking before I posted.

**Comment by essential61** [comment]
I'll delete the releases, I don't think they serve a useful purpose. The head of the master repo is always likely to be the best version to use.
